src/zenml/integrations/mlflow/__init__.py

```diff
@@ -58,7 +58,7 @@ def get_requirements(cls, target_os: Optional[str] = None) -> List[str]:
         from zenml.integrations.pandas import PandasIntegration
 
         reqs = [
-            "mlflow>=2.1.1,<3",
+            "mlflow>=2.1.1,<2.17.0",
```

Comment:
why do we do this again>

Reply:
because it was causing an issue something changed in mlflow and I had to add this because with this PR if mypy failed no test will run